client/web/compose/src/components/ModuleFields/Configurator/Select.vue

```diff
@@ -140,6 +140,16 @@ export default {
     isNew () {
       return this.module.moduleID === NoID || this.field.fieldID === NoID
     },
+
+    defaultFieldOptions () {
+      const field = this.moduleDefaultState.fields.find(({ fieldID }) => fieldID === this.f.fieldID)
```

Comment:
For all new (not yet saved fields) the fieldID will be the same ('0').
So you'll need to make sure you're not returning the wrong options here.